web/public/mgmt/config.yml

```diff
@@ -1233,19 +1233,21 @@ collections:
   - name: "funding-opportunities"
     label: "Fund Opps - Content"
     folder: "content/src/fundings"
-    summary: "/{{urlSlug}} - {{name}}"
+    summary: "[{{agency[0]}}] - {{name}}"
```

Comment:
dust - should we use displayName instead of name?

Reply:
I second that, I think this should be displayname